cassandra_nodetool/tests/conftest.py

```diff
@@ -53,14 +54,24 @@ def cassandra_cluster():
     env['CONTAINER_PORT'] = common.PORT
 
     # We need to restrict permission on the password file
-    os.chmod(os.path.join(common.HERE, 'compose', 'jmxremote.password'), stat.S_IRUSR)
-
+    # Create a temporary file so if we have to run tests more than once on a machine
+    # the original file's perms aren't modified
+    temp_jmx_pass = tempfile.NamedTemporaryFile(dir="/tmp")
```

Comment:
Can you use `temp_dir()` from `datadog_checks_base` and write the file inside ? That way the cleanup will also be taken care of.